### DISCUSSION TITLE
I've forked to start a french translation

### DISCUSSION THREAD

**Discussion by Moosh-be** [openpracticelibrary/openpracticelibrary#2262] [Ideas]
I start to translates  documents in french but perhaps  some gasby expert can help to let translation more than simple added pages.

**Comment by kattekrab** [comment]
Oh my goodness!!  Merci @Moosh-be - this is wonderful - please share a link to your fork - and yes, we definitely need Gatsby help to bring the translations into the site properly!!  There is a strong Japanese translation here: 
https://openpracticelibrary-ja.netlify.app/
github repo - https://github.com/openpracticelibrary-ja

And you've probably seen we already have some spanish translations too.

Apologies for taking so long to respond, I need to sort out why I am not getting notifications!

**Comment by kattekrab** [comment]
@Moosh-be We did experiment with adding internationalisation feature to OPL

 @tim-stasse did some epic work with #2034 but as it did not get reviewed and merged in a timely fashion, the PR got stale, and would have involved lots of rework so I ended up just closing it :(
